node/webpack.config.babel.js

```diff
@@ -0,0 +1,30 @@
+import { join } from 'path';
+
+export default {
+  mode: 'production',
+  output: {
+    path: join(__dirname, '..', 'inst', 'www', 'shiny.fluent'),
+    filename: 'shiny-fluent.js',
+  },
+  resolve: { extensions: ['.js', '.jsx'] },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        use: ['babel-loader'],
+      },
+    ],
+  },
+  externals: {
+    '@/shiny.react': 'modules["@/shiny.react"]',
```

Comment:
Instead of variously named global variables for accessing modules (`React` for `react`, `ReactDOM` for `react-dom`, `window['shiny.react']` for our own `shiny.react` code), I want to introduce the following convention:
1. NPM module `name` is available as `jsmodule['name']`,
2. Custom JS bundled with R package `name` is available as `jsmodule['@/name']`.

Reply:
I'm going to include this info in a vignette for shiny.react. Note: I changed the name from `modules` to `jsmodule` after discovering that the former would prevent custom tiles from working in Leaflet (due to a bug in Leaflet, actually).